### DISCUSSION TITLE
Our own simple branding

### DISCUSSION THREAD

**Discussion by rchiileea** [elax46/custom-brand-icons#540] [Ideas]
![Vintage-Labels4test](https://github.com/elax46/custom-brand-icons/assets/87599807/ca2dd325-7fb1-44ec-b70b-c60be624833a)
![CBI2](https://github.com/elax46/custom-brand-icons/assets/87599807/20173c6c-811c-4c51-aaee-66b3e909570e)
[custom-brand-icons logo.zip](https://github.com/elax46/custom-brand-icons/files/14083043/custom-brand-icons.logo.zip)

**Comment by elax46** [comment]
very nice added as project logo!
